doc/dev/test_proxy_migration_guide.md

```diff
@@ -88,9 +94,8 @@ Docker is a requirement for using the test proxy. You can install Docker from
 [docs.docker.com](https://docs.docker.com/get-docker/). After installing, make sure Docker is running and is using
 Linux containers before running tests.
 
-The test proxy is made available for your tests via a Docker container. Some tests require an SSL connection to work, so
-the Docker image used for the container has a certificate imported that you need to trust on your machine. Instructions
-on how to do so can be found [here][proxy_cert_docs] and need to be followed before running tests.
+Follow the instructions [here][proxy_cert_docs] to complete setup. You need to trust a certificate on your machine in
+order to communicate with the test proxy over a secure connection.
```

Comment:
super nit thing, but consider numbering these steps to perform the one-time setup so people don't miss the cert part after installing docker